lib/main.dart

```diff
@@ -3,7 +3,6 @@ import 'package:flutter/material.dart';
 
 import 'ticker.dart';
```

Comment:
all of the changes in main are just formatting changes from running the dart formatter. There are no code modifications to `main.dart`.